src/Illuminate/Routing/Router.php

```diff
@@ -674,6 +677,21 @@ public static function prepareResponse($request, $response)
     }
 
     /**
+     * Create an illuminate response from a given base response object.
+     *
+     * @param  mixed  $response
+     * @return \Illuminate\Http\Response
+     */
+    private static function createResponseFromBase($response)
+    {
+        return (new Response(
+            $response->getContent(),
```

Comment:
So we lose will lose the benefits of `getOriginalContent` at this point.

I believe we should try first to take `getOriginalContent` if the method exists, then fallback to `getContent`